examples/public/stylesheets/main.less

```diff
@@ -9,8 +10,7 @@
 
 
 // TEMP STUFF:
```

Comment:
Most of the stuff in main.less is hacky temp stuff. I'll move the drag handle out of here, it doesn't belong.